test/functorch/test_ops.py

```diff
@@ -953,6 +952,7 @@ def test_vmapvjp(self, device, dtype, op):
         # skip because this is flaky depending on what the max_norm is!
         skip('nn.functional.embedding', ''),
         skip('to'),  # RuntimeError: required rank 4 tensor to use channels_last format
+        xfail('NumpyExpMarkDirtyAutogradFunction'),  # vmap: inplace into a regular tensor
```

Comment:
Just to check, this is not "calling in-place operation that would mutate a captured Tensor", right?